README.md

```diff
@@ -95,6 +95,12 @@ If you see some JSON with a bunch of resources, it worked! If you encounter any
     - It may be helpful to save this apikey in a secure place like a password manager.
     - Do not hard-code your apikey in your own scripts. Always use an [environment variable](https://www.twilio.com/blog/2017/01/how-to-set-environment-variables.html)
     - You can always re-issue the POST to recover your apikey.
+    - To rotate your apikey (getting a new apikey and making your old key invalid), issue an empty `POST` request to `/api/v1/apikey/rotate` with your apikey included in a header `x-apikey: yourapikey`:
+        ```sh
+        curl -X POST \
+          http://localhost:5000/api/v1/apikey/rotate \
+          -H 'x-apikey: 0a14f702da134390ae43f3639686fe26'
+        ```
```

Comment:
This literally didn't work for me, not sure why...
```
$ curl -X POST   http://127.0.0.1:5000/api/v1/apikey \
   -H 'Content-Type: application/json' \
   -d '{    "email": "aaron.j.suarez1@gmail.com",    "password": "[READACTED]"}'
{
  "apiVersion": "1.0", 
  "data": {
    "apikey": "4e0dec64b7284d04b5c9458d9b58fbd8", 
    "created_at": "2019-07-20 16:23:52", 
    "email": "aaron.j.suarez1@gmail.com", 
    "last_updated": ""
  }, 
  "status": "ok", 
  "status_code": 200
}

$ curl -X POST \
   http://127.0.0.1:5000/api/v1/apikey/rotate \
   -H 'x-apikey: 4e0dec64b7284d04b5c9458d9b58fbd8'
{
  "apiVersion": "1.0", 
  "data": null, 
  "errors": {
    "unauthorized": {
      "message": "You must provide a valid API token in the x-apikey header."
    }
  }, 
  "status": "Unauthorized", 
  "status_code": 401
}
```

Am I doing something wrong?

Reply:
If it was blacklisted, then shouldn't it tell me that it's blacklisted?

Reply:
Probably not a good security practice to tell someone why their auth failed. I mean, it's a poor testing experience, but I do think it's best practice not to leak too much information to a potential hacker. It's like saying "your password is incorrect" instead of "your email or password are incorrect" (the former implies the email address is in the system... information we don't need to provide an attacker)

Reply:
So... it works for me, so I wonder if your key was blacklisted? Although, that first step probably shouldn't have worked either (unless you blacklisted it in between those two commands).

```
$ curl -X POST http://127.0.0.1:5000/api/v1/apikey \
>   -H 'Content-Type: application/json' \
>   -d '{"email": "dprothero@twilio.com", "password": "[REDACTED]"}'
{
  "apiVersion": "1.0",
  "data": {
    "apikey": "9a10218e65d540719891f4559cdef5bf",
    "created_at": "2019-10-25 16:24:09",
    "email": "dprothero@twilio.com",
    "last_updated": "2019-10-26 21:01:04"
  },
  "status": "ok",
  "status_code": 200
}

$ curl -X POST \
>   http://127.0.0.1:5000/api/v1/apikey/rotate \
>   -H 'x-apikey: 9a10218e65d540719891f4559cdef5bf'
{
  "apiVersion": "1.0",
  "data": {
    "apikey": "30df75bbdc9f42ada002dd302945fe54",
    "created_at": "2019-10-25 16:24:09",
    "email": "dprothero@twilio.com",
    "last_updated": "2019-10-31 19:57:41"
  },
  "status": "ok",
  "status_code": 200
}
```

Reply:
Ahhh... so I can repro if I blacklist my key. The /apikey endpoint should probably also return a 401 if your key is blacklisted. I will add that.

Reply:
Oh, so I somehow blacklisted my key but it was still returning my blacklisted key? Interesting.. Fwiw, I didn't mean to blacklist my key, but I did have some trouble running `make migrate` and just repopulated my local database from scratch to get around it. Do you think that would have anything to do with my key being blacklisted?

Reply:
As it turns out, I had to blow away my database and run `make setup` again because `make migrate` failed me for some reason... I'll make an announcement about it in #oc-python-projects when this gets merged in case anyone else runs into that issue